### PR TITLE
elastic_search_visitor: wildcard nest only authors

### PR DESCRIPTION
**Pull request by szymonlopaciuk**
Signed-off-by: Szymon Łopaciuk <szymon@lopaciuk.eu>

### CODE REVIEW

File: inspire_query_parser/visitors/elastic_search_visitor.py [hunk 1]
```diff
@@ -630,11 +630,15 @@ def visit_value(self, node, fieldnames=None):
...
-                                                      fieldnames=bai_fieldnames or fieldnames,
-                                                      analyze_wildcard=True)
+            query = self._generate_query_string_query(
+                node.value,
+                fieldnames=bai_fieldnames or fieldnames,
+                analyze_wildcard=True
+            )
 
-            return generate_nested_query(ElasticSearchVisitor.AUTHORS_NESTED_QUERY_PATH, query)
+            if ElasticSearchVisitor.KEYWORD_TO_ES_FIELDNAME['author'] in fieldnames:
```
Comment (drjova) [inline inspire_query_parser/visitors/elastic_search_visitor.py]: apparently ``fieldnames`` https://github.com/inspirehep/inspire-query-parser/pull/93/files#diff-113310c03456148bafc0ddbf0b89dde5R621 in this function is a string

Comment (drjova) [reply inspire_query_parser/visitors/elastic_search_visitor.py]: it should be ``==`` instead of ``in``

Comment (david-caro) [reply inspire_query_parser/visitors/elastic_search_visitor.py]: :sob:

Comment (szymonlopaciuk) [reply inspire_query_parser/visitors/elastic_search_visitor.py]: Done :+1:

**Pull request merged**